builder/oneandone/config.go

```diff
@@ -23,10 +25,9 @@ type Config struct {
 	SnapshotName   string `mapstructure:"image_name"`
 	DataCenterName string `mapstructure:"data_center_name"`
 	DataCenterId   string
-	Image          string              `mapstructure:"source_image_name"`
-	DiskSize       int                 `mapstructure:"disk_size"`
-	Retries        int                 `mapstructure:"retries"`
-	CommConfig     communicator.Config `mapstructure:",squash"`
```

Comment:
Intentional to lose CommConfig?

Reply:
Yes; as there's already a `Comm                communicator.Config `mapstructure:",squash"`` field line 19